transform/mattermost-analytics/models/intermediate/sales/int_cloud_trial_requests.sql

```diff
@@ -0,0 +1,67 @@
+{{
+    config({
+        "materialized": "table"
+    })
+}}
+
+WITH customers as (
+    SELECT
+        customer_id,
+        email,
+        name,
+        contact_first_name,
+        contact_last_name
+    FROM
+        {{ ref('stg_stripe__customers') }}
+    where
+        created_at >= '2023-04-27' -- only select customers after the release.
+),
+subscriptions as (
+    SELECT
+        subscription_id,
+        cws_installation,
+        cws_dns,
+        customer_id,
+        trial_start_at,
+        trial_end_at,
+        product_id
+    FROM
+        {{ ref('stg_stripe__subscriptions') }}
+),
+products as (
+    SELECT
+        product_id,
+        name,
+        sku
+    FROM
+        {{ ref('stg_stripe__products') }}
+),
+cloud_trial_requests as (
+    select
+        customers.customer_id,
+        customers.email,
+        customers.name as customer_name,
+        customers.contact_first_name as customer_first_name,
+        customers.contact_last_name as customer_last_name,
```

Comment:
These columns are not needed downstream. Perhaps remove them?

If needed, then perhaps we need to discuss about standardizing handling these fields. There are a few "gotcha's":
- `name` cannot be synced to SF.
- First name and last name may be null or empty string. Some special handling might need to be done there.

Reply:
removing them since they are not needed downstream.